openstudiocore/resources/osversion/1_10_4/OpenStudio.idd

```diff
@@ -8276,9 +8276,14 @@ OS:AirLoopHVAC,
        \type object-list
        \required-field
        \object-list ConnectionNames
-  A14; \field Return Air Bypass Flow Temperature Setpoint Schedule Name
+  A14, \field Return Air Bypass Flow Temperature Setpoint Schedule Name
```

Comment:
It does not get translated to E+ at all as pointed out in the first commit message. This is just needed for the baseline automation work